README.md

```diff
@@ -21,22 +21,28 @@ Then to run your tests:
 shpec [shpec_files]
 ```
 
-If you'd like your tests to run automatically when they change, we recommend the [entr](http://entrproject.org/) utility:
+If you'd like your tests to run automatically when they change, we recommend the [entr](
```

Comment:
Trying to keep the raw file readable in my editor.  So I've been experimenting to find out
where the markdown breaks down on github.  This appears to work.